elide-integration-tests/src/test/java/example/CreateButNoUpdate.java

```diff
@@ -27,6 +27,7 @@
     public Long id;
     public String textValue;
 
+    @CreatePermission(expression = "deny all")
```

Comment:
This permission doesn't seem to jive with the class name.

Reply:
We wanted to deny the update permission to the field be it on create or not. Thats why we wanted a @CreatePermission("deny all") expression for forbidding updates on the field specifically.